batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/filter/ExceptionFilter.java

```diff
@@ -0,0 +1,43 @@
+package com.lftechnology.batch7crud.filter;
+
+import com.lftechnology.batch7crud.constant.PageConstant;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**This class handles all the exceptions and sends error code corresponding to the exception message
+ *
+ * @Author Binod Shrestha <binodshrestha@lftechnology.com>
+ * Created on 1/29/16
+ */
+public class ExceptionFilter implements Filter {
+
+  public static final Logger LOGGER = Logger.getLogger(ExceptionFilter.class.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "LOGGER" logger private static final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)